apps/meteor/server/startup/migrations/v309.ts

```diff
@@ -0,0 +1,24 @@
+import { Settings } from '@rocket.chat/models';
+
+import { settings } from '../../../app/settings/server';
```

Comment:
Let's not use the cached setting here, we need to use the model instead

Reply:
Ooh, you're right :man_facepalming: fixed!